### DISCUSSION TITLE
New fresh install get error

### DISCUSSION THREAD

**Discussion by aliqasemzadeh** [laravel/framework#46799] [Q&A] [answered]
Hi
When I install `composer install`
I get the following error:
````
> @php artisan package:discover --ansi
In app.php line 158:
Call to undefined method Illuminate\Support\ServiceProvider::defaultProviders()
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
````

Why do I get this error?
My php version is 8.1

**Comment by Angel5a** [comment, marked as answer]
Looks like framework changes https://github.com/laravel/framework/pull/46786 haven't been published yet, but skeleton https://github.com/laravel/laravel/pull/6159 does.
